src/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import { BrowserRouter as Router } from 'react-router-dom'
 
+import './App.css'
```

Comment:
Move style file to last line

Reply:
if I move it, the styles for the login button disappear for some reason